images/linux/scripts/installers/chromium-browser.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+################################################################################
+##  File:  chromium-browser.sh
+##  Desc:  Installs Chromium browser
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/os.sh
+
+if ! isUbuntu20 ; then
+    exit 0
+fi
+
+#Install chromium browser
+apt install chromium-browser -y
```

Comment:
we've moved all such installations to the toolset here https://github.com/actions/virtual-environments/pull/2376

Reply:
Installation of chromium-browser has been moved to toolset